analyzer/src/main/kotlin/PackageCurationProvider.kt

```diff
@@ -24,22 +24,45 @@ import org.ossreviewtoolkit.model.PackageCuration
 import org.ossreviewtoolkit.model.config.PackageCurationProviderConfiguration
 import org.ossreviewtoolkit.utils.common.ConfigurablePluginFactory
 import org.ossreviewtoolkit.utils.common.Plugin
+import org.ossreviewtoolkit.utils.common.getDuplicates
 
 /**
  * The extension point for [PackageCurationProvider]s.
  */
 interface PackageCurationProviderFactory<CONFIG> : ConfigurablePluginFactory<PackageCurationProvider> {
     companion object {
         val ALL = Plugin.getAll<PackageCurationProviderFactory<*>>()
+        const val ORT_YML_PROVIDER_ID = "ort-yml"
```

Comment:
Like for types, shouldn't we rather align on Camel-Case for IDs, too? Esp. as you default `id` to `type` below.

Reply:
See @fviernau's https://github.com/oss-review-toolkit/ort/pull/6447.